engine/class_modules/sc_death_knight.cpp

```diff
@@ -5098,6 +5094,18 @@ struct epidemic_damage_aoe_t : public death_knight_spell_t
   }
 };
 
+struct epidemic_damage_aoe_t : public death_knight_spell_t
+{
+  epidemic_damage_aoe_t( death_knight_t* p ) :
+    death_knight_spell_t( "epidemic_aoe", p, p -> find_spell( 215969 ) )
+  {
+    background = true;
+    // Since we set epidemic_damage_main_t's impact action to be the impact action from this spell, we end up using
+    // the aoe value from that spell, instead of this one
+    aoe = 0;
```

Comment:
The secondary damage component's aoe should be set to aoe - 1 (or data().max_targets() - 1) and not 0.
The way the spell works is that every flare up on enemies generate a secondary aoe that doesn't hit the enemy that generates it (the "source").
With your current implementation, each flare up would deal damage to the source enemy as well as everything around it *with the higher spelldata coefficient* and then deal the smaller amount to a single target around the source.